pkg/testutil/matcher.go

```diff
@@ -47,8 +47,8 @@ func (cm *EqualMatcher) NegatedFailureMessage(actual interface{}) string {
 		"\nDiff (- Expected, + Actual):\n" + indent(cm.explanation.Error(), 1)
 }
 
-func indent(in string, indentation uint) string {
-	indent := strings.Repeat(format.Indent, int(indentation))
+func indent(in string, indentation int) string {
```

Comment:
Not that we really need it to be a uint in a private function, but what was the lint issue here?

Reply:
It was complaining that the cast may overflow.